src/stories/common.tsx

```diff
@@ -368,7 +368,7 @@ const FrontendEngineWithWarning = ({ id, fieldSchema }: { id: string; fieldSchem
 			<Button.Default
 				onClick={() =>
 					formRef.current?.setWarnings({
-						[id]: "This is a warning message, it is set via `setWarnings()` from Frontend Engine.",
+						[id]: "This is a warning message, <br/>it is set via `setWarnings()` from Frontend Engine.",
```

Comment:
right now all warnings are expected to be plain strings (e.g. [HistogramSlider](http://localhost:6006/?path=/story/field-histogramslider--warning))

if there's a need for html formatting in general then you'll also need to enhance the shared `Warning` component